src/rdbms/rdbms_queries.erl

```diff
@@ -488,29 +498,21 @@ update_roster(_LServer, Username, SJID, ItemVals, ItemGroups) ->
                   end,
                   ItemGroups).
 
-update_roster_sql(Username, SJID, ItemVals, ItemGroups) ->
-    [[<<"delete from rosterusers "
-        "where username=">>, mongoose_rdbms:use_escaped_string(Username), <<" "
-        "and jid=">>, mongoose_rdbms:use_escaped_string(SJID)],
-     [<<"insert into rosterusers("
-        "username, jid, nick, "
-        "subscription, ask, askmessage, "
-        "server, subscribe, type) "
-        " values (">>, join_escaped(ItemVals), ");"],
-     [<<"delete from rostergroups "
-        "where username=">>, mongoose_rdbms:use_escaped_string(Username), <<" "
-        "and jid=">>, mongoose_rdbms:use_escaped_string(SJID), ";"]] ++
-        [[<<"insert into rostergroups(username, jid, grp) "
-            "values (">>, join_escaped(ItemGroup), ");"] ||
-            ItemGroup <- ItemGroups].
-
-roster_subscribe(_LServer, Username, SJID, ItemVals) ->
+%% Dirty hack to filter 'nick' out and overwrite it by mistake; It is a third element
+%% in values list.
+%% TODO: refactor API of this module and replace separate lists of keys and values
```

Comment:
Do you think it's worth doing this refactoring in this PR?

Reply:
It's a fix for a customer so I wanted to avoid extensive changes to save time. I'd prefer to make a separate PR. Actually, I'd prefer to eradicate this module and move all queries to their respective modules.